.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
+*.d.ts
 *.log
+*.tgz
```

Comment:
```suggestion
```

I suggest removing this. I don’t think we need to have a list of *all possible things*.
A list that ignores certain files generated by the build though

Reply:
> I don’t think we need to have a list of _all possible things_.

I totally agree! I believe `.gitignore`  files should only contain file related to a project setup, i.e. no editor settings.

IMO `*.tgz` matches this criteria for NodeJS projects, because `npm pack` (or `yarn pack`) will output the package in the project root.

Reply:
Hmm, I almost never use this feature (and if I do, I manually clean it), nobody has run into it before, and the glob is rather implicit (it’s unclear what it’s actually ignoring and why).

But okay, I’m not blocking this!